doc-content/enterprise-only/openshift/migration-ocp3-4-mysql-proc.adoc

```diff
@@ -0,0 +1,129 @@
+[id='migration-ocp3-4-mysql-proc-{context}']
+= Migrating a MySQL database for a {KIE_SERVER}
+
+If your environment in {OPENSHIFT} version 3 includes a {KIE_SERVER} that uses a MySQL database pod, copy the MySQL database content from the old deployment to the new deployment. This action copies the existing process state to the new deployment.
+
+.Prerequisites
+
+* You must have a machine that has network access to both the {OPENSHIFT} version 3 and {OPENSHIFT} version 4 infrastructures.
+* The `oc` command-line client from {OPENSHIFT} version 4 must be installed on the machine. For instructions about installing the command-line client, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.4/html/cli_tools/openshift-cli-oc#installing-the-cli[_CLI tools_] in {OPENSHIFT} documentation.
+* The `mysql` and `mysqldump` client applications provided by MySQL version 8 or later or by MariaDB version 10 or later must be installed.
+
+.Procedure
+
+. Ensure that no web clients and no client applications are connected to any elements of the old and new deployment, including {CENTRAL} and {KIE_SERVERS}.
+. Create a temporary directory and change into this directory.
```

Comment:
Change into this directory? this sounds confusing. Are we changing to the new directory?